### ISSUE TITLE
Parsing bug

### ISSUE THREAD

**Issue by mycaule** [npm/ini#29]
```
[section1]
alice=value1

[section2]          ; spaces behind section2
bob=value2
```

When you decode the file, the result is not what you expect

```
{ section1: { alice: 'value1', '[section2]          ': true, bob: 'value2' } }
```

It should be 

```
{ section1: { alice: 'value1'}, section2: {bob: 'value2'} }
```

**Comment by megastef** [comment]
+1 Sections are not recognized when having whitespace behind section e.g. "[section] "

**Comment by mycaule** [comment]
Same when the space occur before the section.

```
    [section]
alice=value1
```

**Comment by jnawk** [comment]
Was this fixed?  I have this problem in v1.3.5.   Happy if the answer is fix the damn config, just want to know one way or another.

**Comment by mycaule** [comment]
I don't think so.
It's been 6 years already.

I'm a better developper now!
